src/amo/components/AddonReviewCard/styles.scss

```diff
@@ -49,6 +49,15 @@
       text-decoration: none;
     }
   }
+
+  .AddonReviewCard-delete:first-child .ConfirmButton-default-button {
```

Comment:
Hmm. The `:first-child` looks fragile. How about this instead?

```diff
diff --git a/src/amo/components/AddonReviewCard/styles.scss b/src/amo/components/AddonReviewCard/styles.scss
index 07fb4a5c2..fd8865c0a 100644
--- a/src/amo/components/AddonReviewCard/styles.scss
+++ b/src/amo/components/AddonReviewCard/styles.scss
@@ -50,7 +50,7 @@
     }
   }
 
-  .AddonReviewCard-delete:first-child .ConfirmButton-default-button {
+  .AddonReviewCard-ratingOnly.AddonReviewCard-verticalButtons & .ConfirmButton-default-button {
     color: $text-color-default;
 
     &:active,
```

Reply:
I had to add a `/* stylelint-disable */` comment for this, because prettier keeps "fixing" it, and then stylelint complains. :-P